autospec/files.py

```diff
@@ -254,8 +254,6 @@ def push_file(self, filename):
             (r"^/sbin/", "bin"),
             (r"^/bin/", "bin"),
             (r"^/usr/lib/python3.*/", "python3", "/usr/lib/python3*/*"),
-            (r"^/usr/lib/python2.*/", "legacypython", "/usr/lib/python2*/*"),
-            (r"^/usr/lib64/python.*/", "python", "/usr/lib64/python*/*"),
```

Comment:
Was this mapping ever used?

Reply:
My memory is we originally only had a single mapping and so it was the default and we shortly there after needed both. I'd need to dive through the archives though to be sure.